src/list-data/conversion-types.ts

```diff
@@ -37,7 +37,7 @@ export const ConversionTypes: ConversionTypesI[] = [
   {
     desc: 'Benefit company to a limited company',
     text: 'Benefit company to a limited company',
-    entity_type_cd: EntityType.CR,
+    entity_type_cd: EntityType.BEN,
```

Comment:
I wonder if you should use `CommonMixin::entityTypeAlternateCode()` to convert from EntityType to CorpTypeCd here (or vice versa)?

I know these code are almost the same thing (EntityType is a subset of CorpTypeCd) but it strikes me as strange that "BEN" or "CorpTypeCd.BENEFIT_COMPANY" isn't used anywhere else in Namerequest -- I think for a BEN NR it actually uses the "BC" code.

Reply:
You can also use ConversionTypes.value to determine the "from" and "to" value.

In this code, the value is NrRequestTypeCodes.CONVERT_CORP, which means "from BEN to LTD".

What do you think?

Reply:
As discussed, revert this and maybe use the request type code to check "from" and "to" entity type (since the request type codes have a unique from and to).

Reply:
I see you've added a variable for "original cd". That's fine by me for now.